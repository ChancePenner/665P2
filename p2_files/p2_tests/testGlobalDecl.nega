int global1;
int & global2;
bool & global3;
void test1;
struct MyStruct & instance;
