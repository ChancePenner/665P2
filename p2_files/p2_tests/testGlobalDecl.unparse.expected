int global1;
int & global2;
bool global3;
bool & global4;
void whatever;
void global5;
struct hello myfriend;
struct hello & myBestFriend;
